src/fides/api/ops/api/v1/scope_registry.py

```diff
@@ -184,6 +186,7 @@
 SAAS_CONFIG_READ = f"{SAAS_CONFIG}:{READ}"
 
 SAAS_CONNECTION_INSTANTIATE = f"{CONNECTION}:{INSTANTIATE}"
+CONNECTOR_TEMPLATE_REGISTER = f"{CONNECTOR_TEMPLATE}:{REGISTER}"
```

Comment:
That makes sense to be, I'll put that in for now and back it out if @rsilvery has a differing opinion.

Reply:
@adamsachs , @galvana  this one is tricky since we're going to be moving this functionality to the system info panel shortly. The button will only show up for `contributor` or `owner` after that move or for somebody who has been assigned as the manager for a system. Make sense?

Reply:
Thanks @rsilvery, I'll have this action be for owners only for now and we can add it to contributors once we move the button to its final spot.